Tests/ApplicativeRouterHttpPipelineSupportTests/ApplicativeRouterHttpPipelineSupportTests.swift

```diff
@@ -55,4 +55,20 @@ class ApplicativeRouterHttpPipelineSupportTests: XCTestCase {
 enum Route {
   case home
   case episode(String)
+
+  enum iso {
+    static let home = parenthesize <| PartialIso<Prelude.Unit, Route>(
```

Comment:
the only boilerplate we incur with the new routes is that each enum case of our routing enum needs to have an partial isomorphism `PartialIso<(Tuple, Of, Args), Route>`. the `parenthesize` helper is kind like how we need `curry` in the other applicative router, except that it takes a flat list of tuple arguments and right-parenthesizes them so that we can use it with `<%>`